src/parser/node/cond.rs

```diff
@@ -4,20 +4,21 @@
 //! rule
 //!
 //! ```abnf
-//! cond = "if" block1 "do" block1 ("else" block1)? "end"
+//! cond = "if" block1 "do" block1 ("elif" block1 "do" block1) ("else" block1)? "end"
 //! ```
 //!
-//! Thus, `else` blocks are optional and are represented as empty [`Block`]s inside the
+//! Thus, `elif` and `else` blocks are optional and are represented as empty [`Block`]s inside the
```

Comment:
plz remove the "and `else`". That's a leftover of some changes I did and forgot to update the docs :/